interface/forms/patient_care_plan/new.php

```diff
@@ -0,0 +1,195 @@
+		<style>
+			table, th, td {
+			  border: 0px;
+			  border-collapse: collapse;
+			}
+			th, td {
+			  padding: 5px;
+			  text-align: left;    
+			}
+			.col-sm-6 ,.col-md-3{
+				padding-left: 3px;
+				padding-right: 3px;
+			}
+			input:focus, textarea:focus, select:focus {
+				border-color: #ca1278 !important;
+				border-width: 2px !important;
+			}
+		</style>
+	</head>
+<body>
+	<form method=post action="<?php echo $rootdir;?>/forms/patient_care_plan/save.php?mode=new" name="my_form" onsubmit="beforeSubmit(); return top.restoreSession()">
+		<div class="row">
+			<div class="col-md-4">
+				<!-- Save/Cancel buttons -->
+				<input type="submit" id="save" class="btn btn-success" value="<?php echo xla('Save'); ?>"> &nbsp;
+				<input type="button" id="dontsave" class="deleter btn btn-danger" value="<?php echo xla('Cancel'); ?>"> &nbsp;
+			</div>
+		</div>
+		
+		<table style="width:100%">
+			<thead>
+				<tr>
+				    <th>Issue #</th>
+				    <th>Key Issue <span><a href="#" ><span title="add issue" onclick="addIssue()" style="color: #337ab7" class="glyphicon glyphicon-plus-sign"></span></a></span></th> 
+		 	 	</tr>
+			</thead>
+
+			<tbody class="addIssue">
+				<tr id="row_1">
+				  	<td>
+				  		<mark>1</mark>
+				  		<a href="#" ><span style="color: #337ab7" onclick="addIntervention(1)" class="glyphicon glyphicon-plus-sign" title="Add Interventions"></span> </a>
+				  		<a href="#"><span style="color: red" onclick="deactivateIssue(1)" class="glyphicon glyphicon-ban-circle" title="Deactivate Issue"></span> </a> 
+				  	</td>
+				  	<td><input type="text" name="issue[]" placeholder="key issue"></td>
+				  	<input type="hidden" class="status_1" name="status[]" value="1">
+				  	<input type="hidden"  name="count[]" value="1">
+				  	<td>
+				      <table>
+				      	<thead>
+				      		<tr>
+							   <th>Intervention</th>
+							   <th>Outcome</th> 
+							   <th>Goal</th>
+							   <th>Progress</th>
+							</tr>
+				      	</thead>
+				      	<tbody class="intervention_1">
+				      		<tr>
+					          <td><textarea name="intervention_1[]"></textarea></td>
+					          <td><textarea name="outcome_1[]"></textarea></td>
+					          <td><textarea name="goal_1[]"></textarea></td>
+					          <td><textarea name="progress_1[]"></textarea></td>  
+					        </tr>
+				      	</tbody>   
+				      </table>
+				    </td>    
+			  </tr>
+			</tbody>
+		  
+		</table>
+		<textarea hidden="hidden"   name="Interventions"></textarea>
+		<textarea hidden="hidden" name="Outcome"></textarea>
+		<textarea hidden="hidden" name="Goal"></textarea>
+		<textarea hidden="hidden"  name="Progress" ></textarea>
+
+	</form>
+
+	<script type="text/javascript">
+		var count = 1; 
+		function addIntervention(currentIssueNumber ) {
+			let intervention = createInterventionHtml(currentIssueNumber);
+			$(`.intervention_${currentIssueNumber}`).prepend(intervention);
+		}
```

Comment:
In this file i get an akward behaviour cant delete a row when added
![Screen Shot 2019-09-02 at 4 05 31 AM](https://user-images.githubusercontent.com/16350814/64087960-1faf7a80-cd37-11e9-8ed8-1c611ce24b86.png)

And also is that how it intended to be a single key issue for all of them?

Reply:
Saving patient care plan form does not complete but when u refresh the UI it shows it saved, you might want to check that out 
![Screen Shot 2019-09-02 at 4 08 59 AM](https://user-images.githubusercontent.com/16350814/64088157-12df5680-cd38-11e9-9acc-62306d28f7c1.png)

Reply:
> In this file i get an akward behaviour cant delete a row when added
> ![Screen Shot 2019-09-02 at 4 05 31 AM](https://user-images.githubusercontent.com/16350814/64087960-1faf7a80-cd37-11e9-8ed8-1c611ce24b86.png)
> 
> And also is that how it intended to be a single key issue for all of them?

for now specs did not talk about deleting an intervention. But Maybe I could add that.

Reply:
I don't think we're supposed to delete an intervention because the staff may need to refer to it in the future.

Reply:
ok